ina219/ina219.go

```diff
@@ -110,7 +111,7 @@ func (d *Dev) Sense() (PowerMonitor, error) {
 		return PowerMonitor{}, errReadShunt
 	}
 	// Least significant bit is 10µV.
-	pm.Shunt = physic.ElectricPotential(shunt) * 10 * physic.MicroVolt
+	pm.Shunt = physic.ElectricPotential(*((*int16)(unsafe.Pointer(&shunt)))) * 10 * physic.MicroVolt
```

Comment:
Why not int16(shunt) ? No need for unsafe. See https://go.dev/play/p/uUhKx4hdbeR

Reply:
I typically write go at an abstraction level a bit higher, and to be honest, the 0xFFFF result surprised me. Seems perfectly reasonable, to use that instead though.